src/Components/ArticleImage.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { getArticleImage } from "../api";
+
+const ArticleImage = ({ topic }) => {
+    const [image, setImage] = useState(''); 
+    const [isLoading, setIsLoading] = useState(true); 
+
+    const randomIndex = Math.floor(Math.random() * 11); 
+
+    useEffect(() => {
+        getArticleImage(topic)
+        .then(({ data }) => {
+            setImage(data.results[randomIndex].urls.raw); 
+            setIsLoading(false); 
+        })
+    }, [])
+
+    if (isLoading) return <p>Loading!</p>
+
+    return (
+        <img src={image} alt='' className="article-image"/>
+    )
```

Comment:
Alt tags are very important for accessibility. Ideally you would want to use images where you can determine their contents, but in this case even using the `topic` prop in some form for the alt tag would be an improvement on nothing